braindecode/datasets/base.py

```diff
@@ -16,6 +16,8 @@
 import pandas as pd
 
 from torch.utils.data import Dataset, ConcatDataset
+from braindecode.augmentation.transform_class import Transform
+from braindecode.augmentation.transforms import identity
```

Comment:
I feel that defining a public identity function that does `lambda x: x` is an overkill